client/src/views/K8s.vue

```diff
@@ -123,6 +123,19 @@
                   :key="head"
                 >
                   {{ head }}
+
+                  <v-tooltip
+                    v-if="head === 'Yggdrasil IP'"
+                    text="install Yggdrasil through yggdrasil.com"
```

Comment:
the correct link is https://yggdrasil-network.github.io

should be `visit https://yggdrasil-network.github.io/installation.html to get connected to yggdrasil network`